packages/documentation/.storybook/styles/manager.scss

```diff
@@ -96,6 +96,10 @@
   }
 }
 
+#storybook-explorer-tree [data-item-id^='health'] {
+  display: none;
+}
+
```

Comment:
Can you nest the attribute selector in the #storybook-explorer-tree selector above?